src/python/deepgnn/graph_engine/snark/converter/process.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Conversion functions to internal binary format."""
+import multiprocessing as mp
+import typing
+import platform
+from typing import Optional
+
+if platform.system() == "Windows":
+    from multiprocessing.connection import PipeConnection as Connection  # type: ignore
+else:
+    from multiprocessing.connection import Connection  # type: ignore
+
+import deepgnn.graph_engine.snark.converter.writers as writers
+from deepgnn.graph_engine.snark.decoders import (
+    DecoderType,
+    Decoder,
+    JsonDecoder,
+    TsvDecoder,
+)
+
+
+FLAG_ALL_DONE = b"WORK_FINISHED"
+FLAG_WORKER_FINISHED_PROCESSING = b"WORKER_FINISHED_PROCESSING"
+PROCESS_PRINT_INTERVAL = 1000
+
+
+class _NoOpWriter:
+    def add(self, _: typing.Any):
+        return
+
+    def close(self):
+        return
+
+
+def converter_process(
```

Comment:
This function doesn't look private despite of the title of this PR.

Reply:
I moved it out of the same file as MultiWorkersConverters and into the folder where there is no other user facing code.

Reply:
Anyone can access this function via
```python
from deepgnn.graph_engine.snark.converter.process import converter_process
```
This is a public function, because there is [no leading underscore](https://peps.python.org/pep-0008/#method-names-and-instance-variables)

Reply:
I updated the PR description and changelog, the main goal is to remove it from the MultiWorkersConverter file since the function is only used by Dispatchers.